events/spi/src/main/java/org/projectnessie/events/spi/NessieConfiguration.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.events.spi;
+
+import java.util.Map;
+import org.immutables.value.Value;
+
+/** Configuration of the Nessie server. */
+@Value.Immutable
+public interface NessieConfiguration {
```

Comment:
This is similar to `org.projectnessie.model.NessieConfiguration`.

I gave up on the initial suggestions to include things like: 

1. server address and hostname: the subscriber can compute it by themselves;
2. server exact version: for security reasons;
3. more configuration options such as: metrics/tracing/authz/authn enabled, etc.: for security reasons as well.
4. repository information: cannot be supplied here since Nessie is mono-repo for now, but is meant to be multi-repo in the future. The info provided here must be server-wide.

Let me know if you think something is missing.